fuse/eval/metrics/classification/metrics_classification_common.py

```diff
@@ -145,6 +145,28 @@ def __init__(
         super().__init__(pred, target, metric_func=auc_roc, class_names=class_names, **kwargs)
 
 
+class MetricAUCROCmultLabel(MetricMultiClassDefault):
+    """
+    Compute auc roc (Receiver operating characteristic) score using sklearn (one vs rest)
+    """
+
+    def __init__(
+        self,
+        pred: str,
+        target: str,
+        max_fpr: Optional[float] = None,
+        average: str = "micro",
+        **kwargs,
+    ):
+        """
+        See MetricMultiClassDefault for the missing params
+        :param max_fpr: float > 0 and <= 1, default=None
+                        If not ``None``, the standardized partial AUC over the range [0, max_fpr] is returned.
+        """
+        auc_roc = partial(MetricsLibClass.auc_roc_mult_binary_label, average=average, max_fpr=max_fpr)
+        super().__init__(pred, target, metric_func=auc_roc, **kwargs)
```

Comment:
To fix your problem, you specify the name of the arguments explicitly cause the order is different.
i.e. (pred=pred, target=target, ...) 
Please also change the name to MetricAUCROCMultLabel (capital M)

Reply:
fixed